### PR TITLE
Rewrite using Material Components

### PR DESCRIPTION
**Pull request by brunohpaiva**
- Switched from [Bootstrap](https://getbootstrap.com/) to [Material Components](https://material.io/develop/web/)

Preview [here](https://hyperium-master.brunohpaiva.com.br)

image
image
image

### CODE REVIEW

File: download/index.html [hunk 1]
```diff
@@ -1,20 +1,11 @@
 <!DOCTYPE html>
 <html lang="en">
-
-<!-- Redirect page
-made with <3 by RDIL for Sk1er
```
Comment (RDIL) [inline download/index.html:5]: how dare you remove my credit

**Review by RDIL — CHANGES_REQUESTED** [review]
:(

File: download/index.html (hunk 1 above)
Comment (brunohpaiva) [reply download/index.html:5]: probably increases the download time of the document for redirect, but if you insist..

Comment (RDIL) [reply download/index.html:5]: comments aren't downloaded by the majority of most modern browsers

**Review by RDIL — APPROVED** [review]

**Comment by Cubxity** [comment]
You should prob minify the files or something

File: download/index.html (hunk 1 above)
Comment (Cubxity) [reply download/index.html:5]: @RDIL comments are loaded with the page

**Comment by RDIL** [comment]
Making the comment 1line should be small enough

**Comment by brunohpaiva** [comment]
> You should prob minify the files or something

Hm, I'll add a basic node script to minify css and js

**Comment by brunohpaiva** [comment]
Now using Babel 7.2.0 and CleanCSS 4.2

**Review by Cubxity — APPROVED** [review]

**Pull request merged**